perf-baseline/reporter/main.go

```diff
@@ -55,6 +57,8 @@ var (
 func main() {
 	namespace = flag.String("namespace", "l5d-perf", "namespace where the performance test is running")
 	logLevel := flag.String("log-level", log.InfoLevel.String(), "log level, must be one of: panic, fatal, error, warn, info, debug")
+	warmupTime = flag.Uint("warmup-time", 7*60, "Number of seconds to wait before starting to report numbers")
+	reportInterval = flag.Uint("report-interval", 10, "Interval in seconds at which statistics are dumped")
```

Comment:
Should the default be 60 to match the previous behavior of reporting once per minute?